rs/state_layout/src/state_layout/tests.rs

```diff
@@ -814,3 +820,114 @@ fn can_add_and_delete_canister_snapshots(
         check_snapshot_layout(&checkpoint_layout, &snapshot_ids[(i + 1)..]);
     }
 }
+
+// This will be removed after EXC-1752.
+#[test]
```

Comment:
I'm not sure this test is needed (I don't think we necessarily write such tests for such transformations). But if we want to have it, imo it would make more sense that we have a `CanisterStateBitsV1` and `CanisterStateBitsV2` in order to test what we really want (but that's extra effort which is why I'm not sure we necessarily need the test).

Furthermore, the latter part of the test now tests a case that would not happen with the current code -- the old fields are not going to be unset, not until we remove them in a follow up PR.

Reply:
Test is removed.